test/integration/rules/td-headers-attr/td-headers-attr.html

```diff
@@ -32,6 +32,11 @@
   <td id="self" headers="self">World</td>
 </table>
 
+<table id="fail4">
+  <td id="hdr1">Hello</td>
+  <td headers="hdr1">World</td>
+</table>
```

Comment:
I think we need to consider what the role attribute does, not just the node name. We'll need to do screen reader testing to learn what works and what doesn't. Here are the scenarios I'm thinking of:

```html
<table>
  <td role="rowheader" id="hdr1">Hello</td>
  <td headers="hdr1">World</td>
</table>
```

```html
<table>
  <th role="cell" id="hdr1">Hello</td>
  <td headers="hdr1">World</td>
</table>
```

```html
<table>
  <th role="button" id="hdr1">Hello</td>
  <td headers="hdr1">World</td>
</table>
```